src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,110 @@
+// Avatar Component
+
+import { cn } from "@/utils";
+import { VariantProps } from "class-variance-authority";
+import React, { ComponentProps } from "react";
+import {
+  avatarFallbackStyle,
+  avatarImageStyle,
+  avatarStyles,
+} from "./index.style";
+
+// Avatar
+
+type CustomAvatarProps = {
+  children?: React.ReactNode;
+  size?: "sm" | "md" | "lg";
+};
+
+type AvatarProps = ComponentProps<"div"> &
+  CustomAvatarProps &
+  VariantProps<typeof avatarStyles>;
+
+export const Avatar: React.FC<AvatarProps> = ({
+  children,
+  size,
+  className,
+  ...props
+}) => {
+  return (
+    <div className={cn(avatarStyles({ size }), className)} {...props}>
+      {children}
+    </div>
+  );
+};
+
+// Avatar Image
+
+type CustomAvatarImageProps = {
+  src: string;
+  size?: "sm" | "md" | "lg";
+  alt?: string;
+  fallback?: React.ReactNode;
+};
+
+type AvatarImageProps = ComponentProps<"img"> &
+  CustomAvatarImageProps &
+  VariantProps<typeof avatarImageStyle>;
+
+export const AvatarImage: React.FC<AvatarImageProps> = ({
+  src,
+  size = "md",
+  alt = "Alternative Text",
+  fallback,
+  className,
+  ...props
+}) => {
+  if (!src || src === "") {
+    return <AvatarFallback size={size} alt={alt} fallback={fallback} />;
```

Comment:
@svipulc can this be sufficient?

```suggestion
  if (!src) {
    return <AvatarFallback size={size} alt={alt} fallback={fallback} />;
```